lib/hanami/application.rb

```diff
@@ -1,12 +1,12 @@
 # frozen_string_literal: true
 
 require "dry/system/container"
+require "hanami/application/settings"
 require "hanami/configuration"
 require "pathname"
 require "rack"
 require_relative "slice"
 require_relative "application/autoloader/inflector_adapter"
-require_relative "application/settings"
```

Comment:
No particular reason besides that I tend to use `require_relative` only when `require` doesn't work :slightly_smiling_face: Do you prefer to switch it back?

Reply:
For me, I feel nicer if we don't expose that we are relying on the filesystem structure, even if it's the same under the hood.

Reply:
@waiting-for-dev I've taken to the pattern of using require_relative for requires to neighbouring files in the gem. It's a signal about which requires are external vs internal, and it's also a minor performance boost (see e.g. [this PR](https://github.com/rails/rails/pull/29638) where the entire Rails codebase was ported over.

In a repo like this one it's somewhat inconsistent since it's only in place for the areas I've worked on recently, but I hope to port everything over with time.

Reply:
So let's keep it consistent! Switched back! :slightly_smiling_face: 

In fact, it's not that I wanted to replace your `require_relative`, but I didn't notice it, and I added my `require` on top. Later I saw the `require_relative`, and I thought that I had added it myself before :sweat_smile: